plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/RegisterTableProcedure.java

```diff
@@ -195,6 +195,8 @@ private static String normalizeS3Uri(String tableLocation)
     {
         // Normalize e.g. s3a to s3, so that table can be registed using s3:// location
         // even if internally it uses s3a:// paths.
-        return tableLocation.replaceFirst("^s3[an]://", "s3://");
+        String normalizedSchema = tableLocation.replaceFirst("^s3[an]://", "s3://");
+        // Remove trailing slashes so that test_dir is equal to test_dir/
```

Comment:
`slashes` -> `slash`.

FYI, the previous implementation `replaceAll("/+$", "")` remove multiple slashes, which was incorrect for blob storage.

Reply:
`stripTrailingSlash` strips all trailing slashes, which is the same what is done when building `TableMetadata` during table creation.

Reply:
Oops. I misunderstood how it works. Thanks for the correction.